events/wc_events/wc_race_events.txt

```diff
@@ -513,5 +513,10 @@ WCRAC.5 = {
 			# To avoid recursion
 			# death = { death_reason = death_vanished }
 		}
+
+		# Starts mana regeneration loop
+		trigger_event = {
+			on_action = wc_on_mana_regeneration days = { 1 27 } # So it will not cause a major lag on every 1st of each month
```

Comment:
If the lag is a point, it's probably will be better if the pulse (`wc_on_mana_regeneration`) is started on the birthday.

Also, it will be super good if we can fire the pulse only when the character picks perks or traits that give you mana. But so far, I don't see such a possibility, so it's sad.